packages/webapp/src/components/Form/FormNavigationButtons/index.tsx

```diff
@@ -0,0 +1,72 @@
+/*
+ *  Copyright 2024 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+
+import React from 'react';
+import clsx from 'clsx';
+import { useTranslation } from 'react-i18next';
+import Button from '../Button';
+import styles from './styles.module.scss';
+import { ReactComponent as ChevronRight } from '../../../assets/images/buttons/chevron-right.svg';
+
+export interface FormNavigationButtonsProps {
+  isFinalStep?: boolean;
+  isDisabled?: boolean;
+  contextualContent?: React.ReactNode | string;
+  informationalText?: string;
+  onCancel?: () => void;
+  onContinue?: () => void;
```

Comment:
When are these not expected to be passed...?

Reply:
Almost defeats the purpose of TypeScript there!!